build/automation/var/profile/dev.mk

```diff
@@ -80,3 +80,10 @@ COGNITO_USER_POOL_CLIENT_ID := $(or $(COGNITO_USER_POOL_CLIENT_ID), )
 COGNITO_USER_POOL_ID := $(or $(COGNITO_USER_POOL_ID), )
 
 POSTCODE_MAPPING_SERVICE_URL := https://uec-dos-api-pc-dev-uec-dos-api-pc-ingress.k8s-nonprod.texasplatform.uk/api
+
+#Authentication login endpoint is set for fuzzy search at the moment. This should be configured to point authentication service api
+AUTH_LOGIN_URL := https://uec-dos-api-sfs-dev-uec-dos-api-sfs-service.k8s-nonprod.texasplatform.uk
+AUTH_LOGIN_URI := /authentication/login
+#admin access is set for now and this should be changed accordingly with user who has access to the api
+POSTCODE_MAPPING_USER := service-finder-admin@nhs.net
```

Comment:
Talk to @daonhs about who is going to merge this in first. One of you guys will need to update these details once the new cognito users have been added